packages/shared/src/lib/user.ts

```diff
@@ -43,6 +43,11 @@ export interface UserProfile {
   acceptedMarketing?: boolean;
 }
 
+export type UserShortProfile = Pick<
+  LoggedUser,
```

Comment:
Please check if it makes more sense to use the `UserProfile` here

Reply:
It seems the `UserProfile` doesn't have the image property, but then, the `PublicProfile` has it, I will change the current one with `PublicProfile` as it feels more appropriate.